whereby/android-sdk-demo
# Add room helper class

Adds a room helper class for the shared logic between RoomActivityExample and RoomFragmentExample.

**Test plan**
1. Setup the room URL string property in the `RoomHelper` class.
2. Run the app. From the main screen, press the first button "Start using WherebyRoomActivity" and join the room. In the Logcat tab (at the bottom), you can filter the logs by `WherebyRoomEvent` and should see the room events being printed (see screenshot).
3. Run the app. From the main screen, press the second button "Start using WherebyRoomFragment" and join the room. In the Logcat tab (at the bottom), you can filter the logs by `WherebyRoomEvent` and should see the room events being printed (see screenshot). 
3.b. While in the room, press the microphone and camera external buttons (not the ones in the room fragment). Confirm the external button color is updated correctly. 

**Screenshots**
<img width="1440" alt="Screenshot 2022-06-30 at 13 19 06" src="https://user-images.githubusercontent.com/7428931/176665176-dab28756-9649-47d8-95d7-a623bc13c95e.png">
